tests/functional/bin/run_sql_task_test.py

```diff
@@ -10,33 +10,39 @@
 
 
 class TestRunSQLTask:
+    def test_sanity(self, environ):
+        self.run_task(environ, "hello_world")
+
     # We use "db_engine" here to ensure the schema is created
     @pytest.mark.usefixtures("db_engine")
     # If you want to run these tests, you first need to start services in H
     # and run the `report/create_from_scratch` task there.
     @pytest.mark.xfail(reason="Requires reporting tasks to be run in H")
     def test_reporting_tasks(self, environ):
-        for report_path in (
+        for task_name in (
```

Comment:
I vaguely remember writing these as a `pytest.mark.parametrize` once but here you'll have to mark all of them as xfail except "hello_world" which might be a bit less clear.

Reply:
This is a bit different as we don't want the DB cleared between runs. I'm not sure it actually is, but we explicitly want to run them back to back.